spec/index.bs

```diff
@@ -570,18 +570,16 @@ interface mixin GPUObjectBase {
 };
 </script>
 
-<div algorithm>
-    <div data-timeline=content>
-        To <dfn abstract-op>create a new WebGPU object</dfn>({{GPUObjectBase}} |parent|,
-        interface |T|, {{GPUObjectDescriptorBase}} |descriptor|)
-        (where |T| extends {{GPUObjectBase}}):
-
-        1. Let |device| be |parent|.{{GPUObjectBase/[[device]]}}.
-        1. Let |object| be a new instance of |T|.
-        1. Set |object|.{{GPUObjectBase/[[device]]}} to |device|.
-        1. Set |object|.{{GPUObjectBase/label}} to |descriptor|.{{GPUObjectDescriptorBase/label}}.
-        1. Return |object|.
-    </div>
+<div algorithm data-timeline=content>
+    To <dfn abstract-op>create a new WebGPU object</dfn>({{GPUObjectBase}} |parent|,
+    interface |T|, {{GPUObjectDescriptorBase}} |descriptor|)
+    (where |T| extends {{GPUObjectBase}}) run the following steps on the [=content timeline=]:
```

Comment:
Done. Updated some pre-existing text to match this pattern as well. There are several instances that say something like "run the following steps on the [=device timeline=] of |device|" and I've left those alone because I feel like that's less ambiguous.